components/sandbox/tiers/src/konflux-eaas/cluster.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: konflux-eaas-cluster-resources
```

Comment:
Metadata should have empty object like in here https://github.com/redhat-appstudio/infra-deployments/blob/main/components/sandbox/tiers/src/appstudio/cluster.yaml#L3

Reply:
Fixed.